cal/cal/static/js/app.js

```diff
@@ -3,32 +3,120 @@ var analyticsApp = angular.module('analyticsApp', ['nvd3', 'ui.calendar']);
 analyticsApp.controller('LoggedInCtrl', function LoggedInController($scope) {
 });
 
-// Controller to generate graph data from chart directive, cumulative tags
 analyticsApp.controller('TagsCtrl', function($scope, $http){
   var url = '/v1/tags.json';
+  var tagUrl = '/v1/tags/';
```

Comment:
Hmm, so we now have both `/v1/tags/`(1) and `/v1/tags.json`(2).

I think the cleanest way to write these is something like
`tagUrl = "/v1/tags";`
and in the $http in line 14 do
`tagUrl + ".json"`
and whenever we need to access the TagDetail endpoint do
`tagUrl + "/[id]"`

Note that `/v1/tags` is the same as `/v1/tags/` so probably better to leave the last slash out. Thoughts?